src/library/Uno.Themes.WinUI.Markup/Themes/Slider.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class Slider
+		{
+			public static class Resources
+			{
+				public static class Default
+				{
+					public static class Typography
+					{
+						public static class FontFamily
+						{
+							[ResourceKeyDefinition(typeof(Microsoft.UI.Xaml.Media.FontFamily), "SliderFontFamily")]
+							public static ResourceValue<Microsoft.UI.Xaml.Media.FontFamily> Default => new("SliderFontFamily", true);
+						}
+
+						public static class FontSize
+						{
+							[ResourceKeyDefinition(typeof(double), "SliderFontSize")]
+							public static ResourceValue<double> Default => new("SliderFontSize", true);
+						}
```

Comment:
No need for a Default property for these, the `FontFamily` and `FontSize` should be direct properties for `Typography`

Reply:
Removed